roles/cinder-common/tasks/ceph_integration.yml

```diff
@@ -1,4 +1,12 @@
 ---
+- name: Install ceph-common
+  apt: pkg={{ item }} state=latest
```

Comment:
agree, but for hotfix we should be fine. Cloud Archive repo is for mitaka and ceph pkgs would not be bumped up there. On master we should fix this all with better approach.